cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -296,8 +296,8 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
             }
         }
 
-        if (evaluatorRun.violations.isNotEmpty()) {
-            println("Rule violations found.")
+        if (evaluatorRun.violations.any { violation -> violation.severity != Severity.HINT }) {
+            println("Rule violations of severity 'warning' or 'error' found.")
```

Comment:
If we do filtering (sort of) on the violations now anyway, I wonder whether we could also `partition {}` the violations into hints and non-hints, and then use the non-hints list here to exactly state which of "errors" and / or "warnings" we had.

Reply:
We could - but we already have a `printSummary` which prints the summary split into errors/warnings/hints, so that information will already be available in the logs (pretty much two to a few lines above this message).

See here: https://github.com/oss-review-toolkit/ort/blob/20f333bfd073dceef5f8ab34b2cb58c2097bdd5a/cli/src/main/kotlin/commands/EvaluatorCommand.kt#L279

If we sort again here, I wonder whether we should just delete the first summary.

Reply:
> We could - but we already have a `printSummary`

Good point. I guess my preferred solution here would be to inline `printSummary` to after writing the output files, and then combine the summary logic with the condition which exit code to use.

Reply:
I inlined the `printSummary`. We could add a message to the return code, but I couldn't think of a good one so I just printed the violation count.